homeassistant/components/obihai/sensor.py

```diff
@@ -11,16 +11,14 @@
     SensorDeviceClass,
     SensorEntity,
 )
-from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers import issue_registry as ir
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
-from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
 from .connectivity import ObihaiConnection
-from .const import DEFAULT_PASSWORD, DEFAULT_USERNAME, DOMAIN, OBIHAI
+from .const import DEFAULT_PASSWORD, DEFAULT_USERNAME, OBIHAI
 
 SCAN_INTERVAL = timedelta(seconds=5)
 
```

Comment:
PLATFORM_SCHEMA can also be removed (and all the relevant imports above for it).

Reply:
Thanks 👍